l2_naming/naming_4.md

```diff
@@ -9,21 +9,22 @@ if you don't understand some word, look for it in Google
 
 Table Vehicle
 
-*   brand
-*   model
-*   vin
-*   updated
 
-*   manufacture
-*   max_speed
-*   power
-*   emissions
-*   photo
+  brand (varchar)
+  model (varchar)
+  vin (varchar)
+  updated (date)
+
+  manufacture (varchar)
+  max_speed (int)
+  power (varchar)
+  emissions (char)
+  photo (blob)
 
 Table Owner
 
-*   identifier
-*   age
-*   first
-*   last
-*   purcharse
\ No newline at end of file
+  identifier (int)
+  age (int)
+  first (varchar)
+  last (varchar)
+  purcharse (date)
```

Comment:
Como viches na corrección, faltaría por unidades de medida e máis indicacións do que son: purcharsed_at, emissions_ppm, power_cv ...
Aparte: first e last son numerais, deben acompañar substantivos.